tests/test_point_density.py

```diff
@@ -149,6 +160,60 @@ def test_point_density_ppf(scale: Scale):
     assert uniform_dist.ppf(1) == pytest.approx(scale.high, rel=0.1)
 
 
+@pytest.mark.parametrize(
+    "scale", [Scale(-3, 10), LogScale(0.01, 5, 500)],
+)
+def test_point_density_modes(scale: Scale):
+    dist = get_dist_from_scale(scale)
+
+    modes = dist.modes()
+
+    assert len(modes) > 0
+
+    mode_densities = np.array([dist.pdf(mode) for mode in modes])
+
+    assert np.all(mode_densities == mode_densities[0])
+
+    mode_density = mode_densities[0]
+
+    # It's important to test against dist.to_arrays()
+    # instead of e.g. dist.true_xs.
```

Comment:
TBC, with no arguments to `to_arrays`, the exported xs should be the same as `true_xs`, if I'm not mistaken?

Reply:
Yes, the tests still seem to pass if I switch it